lib/block-supports/typography.php

```diff
@@ -248,6 +249,11 @@ function gutenberg_get_typography_value_and_unit( $raw_value, $options = array()
 		return null;
 	}
 
+	// Converts numbers to pixel values by default.
+	if ( is_numeric( $raw_value ) ) {
```

Comment:
I think the docblock should be updated for this function to indicated that an `int` is possible for the `$raw_value`.

Reply:
Since `wp_get_typography_value_and_unit` will be parsing values from theme.json, I think this function should gaurd against anything other than a number and a string for the `$raw_value`.  Also, the tests should add some unhappy paths to test for these unacceptable values, like `{}` or `[]`.

Reply:
Changes reflected in the backport PR https://github.com/WordPress/wordpress-develop/pull/3428. @dream-encode I added you as a reviewer to that PR.

Reply:
Thanks folks! 🙏 I'll make sure these changes make it back to Gutenberg.